src/cript/nodes/supporting_nodes/file.py

```diff
@@ -94,7 +94,7 @@ class File(PrimaryBaseNode):
     | Attribute       | Type | Example                                                                                               | Description                                                                 | Required |
     |-----------------|------|-------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------|----------|
     | source          | str  | `"path/to/my/file"` or `"https://en.wikipedia.org/wiki/Simplified_molecular-input_line-entry_system"` | source to the file can be URL or local path                                 | True     |
-    | type            | str  | `"logs"`                                                                                              | Pick from [CRIPT File Types](https://app.criptapp.org/vocab/file-type/)          | True     |
+    | type            | str  | `"logs"`                                                                                              | Pick from [CRIPT File Types](https://app.criptapp.org/vocab/file_type/)          | True     |
```

Comment:
there are couple of these changes that chane `-` to `_`.
Is that intentional?

Reply:
yep, I was using [Xenu](https://home.snafu.de/tilman/xenulink.html#Download) to catch any broken links, but it doesn't do well when it tries to go on CRIPT UI, so those were less caught. Recently, I found that the old frontend had urls as `file-type` and the new one has them as `file_type` and I fixed any that I could find